test/integration/controller_test.go

```diff
@@ -748,6 +748,7 @@ func newControllerTestTestController(ct *controllerTest) (
 	// create a fake kube client
 	fakeKubeClient := &fake.Clientset{}
 	fakeKubeClient.Lock()
+	prependGetNamespaceReaction(fakeKubeClient, "test-namespace")
```

Comment:
use const `testNamespace`

it's already defined https://github.com/kubernetes-sigs/service-catalog/blob/f3b2f3ff22a819b1f6bf3915d6ab32315433d94e/test/integration/controller_test.go#L59